### PR TITLE
do not include {} boto filter by default

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
Depends-On: https://github.com/ansible-collections/amazon.aws/pull/480

* When no filter: specified, do not include an empty dict in the list of
boto filters because it effectively matches any host.

Closes: #457
Closes: #452

##### ISSUE TYPE

- Bugfix Pull Request

### CODE REVIEW

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by tremble** [comment]
recheck

File: changelogs/fragments/include_filters_with_filter.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+- "inventory - ``include_filters`` won't be ignored anymore if ``filters`` is not set (https://github.com/ansible-collections/amazon.aws/pull/456)."
```
Comment (tremble) [inline changelogs/fragments/include_filters_with_filter.yaml:3]:
```suggestion
- "inventory - ``include_filters`` won't be ignored anymore if ``filters`` is not set (https://github.com/ansible-collections/amazon.aws/issues/457)."
```

**Review by tremble — APPROVED** [review]
Just one minor niggle, but otherwise LGTM.

**Comment by goneri** [comment]
FTR, We've got a problem with the intermediate registry: https://github.com/ansible/ansible-zuul-jobs/issues/1082

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**